hash/megadriv.xml

```diff
@@ -35026,6 +35100,39 @@ Has missing sound samples (i.e. tire screech, "time bonus" on checkpoints)
 	</software>
 
 
+	<!--
+		Games for the HeartBeat Personal Trainer.
+		These require use of the HeartBeat Catalyst fitness sensor.
+	-->
+
+	<software name="outbackj" supported="no">
+		<description>Outback Joey (HeartBeat Personal Trainer)</description>
+		<year>1993</year>
+		<publisher>HeartBeat Corporation</publisher>
+		<info name="usage" value="Requires HeartBeat Catalyst fitness sensor." />
+		<part name="cart" interface="megadriv_cart">
+			<dataarea name="rom" width="16" endianness="big" size="1048576">
+				<rom name="outback joey.bin" size="1048576" crc="10abd93f" sha1="ec2eef751bc8515bc99293d96f833270916c9af7" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="pga2hb" cloneof="pga2" supported="no">
+		<description>PGA Tour Golf II (HeartBeat Personal Trainer)</description>
+		<year>1995</year>
+        <publisher>HeartBeat Corporation</publisher>
+		<info name="usage" value="Requires HeartBeat Catalyst fitness sensor." />
+		<part name="cart" interface="megadriv_cart">
+			<feature name="slot" value="rom_sram" />
+			<dataarea name="rom" width="16" endianness="big" size="1048576">
+				<rom name="pga_tour_golf_u.bin" size="1048576" crc="3eff8b37" sha1="c9c18b389a9afc5f30ab5d457f8f72e1966ff1e6" />
+			</dataarea>
+			<dataarea name="sram" size="16384">
+			</dataarea>
+		</part>
+	</software>
+
```

Comment:
Specific input usage doesn't mean that these two entries should be treated in a special way, please respect the alphabetical order.

Reply:
Ok, I've just moved them to their proper alphabetical order.